skimage/measure/_regionprops.py

```diff
@@ -101,6 +102,7 @@ def __init__(self, slice, label, label_image, intensity_image,
         self.label = label
 
         self._slice = slice
```

Comment:
Shouldn't we remove `_slice` then?

Reply:
Also, isn't it confusing to use a reserved word here in several places?

Reply:
@stefanv I'm all ears for an alternative attribute name. Note that the `slice` in the `__init__` is old, and annoying to deprecate. `self.slice` is *not* very confusing, though, only a little bit confusing.

@soupault even though `._slice` is private, I'm concerned some of our users might have accessed it. So I thought we might as well keep it around, because it's not very costly to do so.

Reply:
@jni I see.
As for the name, maybe something like `region_slice` or `slice_region`?

Reply:
self.slice is probably OK.  I'm more concerned with cases where the built-in gets clobbered.  `region_slice` or `slice_` work for me.